### DISCUSSION TITLE
How can I download docker folders to local  ?

### DISCUSSION THREAD

**Discussion by acodervic** [microsoft/vscode-docker#3699] [Q&A]
How can I download docker folders to local  ?

Please help me

**Comment by bwateratmsft** [comment]
Converting to a discussion. The Docker extension doesn't have a feature for downloading entire folders, but you can easily use the `docker cp` command. For example:

```sh
docker cp containerId:/path/to/folder /destination/path
```

I have a container running with ID `27d22`, and I want to copy the `/home` directory in the container to the local folder, `.`

```sh
docker cp 27d22:/home .
```

**Comment by acodervic** [comment]
Thank you, i meaning, if there has a button (for download dir) on vscode-docker it would be better.
